packages/node_modules/@webex/redux-module-media/src/actions.js

```diff
@@ -217,6 +217,31 @@ export function listenForCalls(sparkInstance) {
   };
 }
 
+/**
+ * Checks if the machine has at least one audio or video device
+ * @returns {Object} {
+ *    audio: true/false,
+ *    video: true/false
+ *}
+ */
+export function getSupportedDevice() {
+  return Promise.resolve().then(() => {
+    if (!navigator.mediaDevices || navigator.mediaDevices.enumerateDevices === undefined) {
```

Comment:
If we can't enumerate the devices, should we default them to `false` instead? Maybe @taymoork2 can provide some more insight.

Reply:
any insight on this @taymoork2  ?

Reply:
probably best to default to false than take the chance of setting it to true

Reply:
however we probably will start seeing issues with IE11 since enumerateDevices isnt available in ie11

https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/enumerateDevices

Reply:
But currently do we support IE11 at all?

Reply:
I don't think that IE11 has a ton, if not very little, webrtc support
also this is implemented on the SDK

Reply:
done!